apps/dcos_l4lb/src/dcos_l4lb_ipset_mgr.erl

```diff
@@ -0,0 +1,334 @@
+-module(dcos_l4lb_ipset_mgr).
```

Comment:
Should we add a SUITE of tests for ipset at least to cover some basic cases? I think this could serve as a safety net for future changes, especially if done by other folks in the team.

Reply:
Unfortunately, there is no basic case here. The current set of tests are checking that ipset and new iptables rules are not breaking any basic cases. Also I added an integration test for this non-trivial case to dcos repo. Please see test_vip_port_mapping [1].

[1] https://github.com/dcos/dcos/pull/5110/files#diff-89643056102d7647724e7dc94be80843R278